core/jobs/batch_jobs/user_validation_jobs_test.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.batch_jobs.user_validation_jobs."""
+
+from __future__ import annotations
+
+from core.jobs import job_test_utils
+from core.jobs.batch_jobs import user_validation_jobs
+from core.jobs.types import job_run_result
+from core.platform import models
+
+from typing import Final, Type
+
+MYPY = False
+if MYPY:
+    from mypy_imports import user_models
+
+(user_models,) = models.Registry.import_models([models.Names.USER])
+
+
+class GetUsersWithInvalidBioJobTests(job_test_utils.JobTestBase):
+
+    JOB_CLASS: Type[
+        user_validation_jobs.GetUsersWithInvalidBioJob
+    ] = user_validation_jobs.GetUsersWithInvalidBioJob
+
+    USER_ID_1: Final = 'id_1'
+    USER_ID_2: Final = 'id_2'
+    USER_ID_3: Final = 'id_3'
+
+    def test_empty_storage(self) -> None:
+        self.assert_job_output_is_empty()
+
+    def test_user_with_null_bio(self) -> None:
+        user = self.create_model(
+            user_models.UserSettingsModel,
+            id=self.USER_ID_1,
+            email='a@a.com'
+        )
+        user.update_timestamps()
+        self.put_multi([user])
+
+        self.assert_job_output_is([
+            job_run_result.JobRunResult(
+                stderr=f'The id of user is "{self.USER_ID_1}" and its bio '
+                + f'is "{user.user_bio}"'
+            ),
+            job_run_result.JobRunResult(
+                stdout='INVALID SUCCESS: 1'),
```

Comment:
"INVALID SUCCESS" is quite confusing. Better to have it be something like "CountInvalidUserBios SUCCESS: 1"?

Ditto for "USERS SUCCESS: 1" below.

Reply:
Done